modules/katello_repository.py

```diff
@@ -142,8 +153,9 @@ def main():
             product=dict(required=True),
             organization=dict(required=True),
             name=dict(required=True),
-            content_type=dict(required=True),
+            content_type=dict(required=True, choices=['docker', 'ostree', 'yum', 'puppet', 'file', 'deb']),
             url=dict(),
+            docker_upstream_name=dict(),
```

Comment:
Should there be a test, that this is only used in combination with `content_type='docker'`?

Reply:
I guess so, but only a pleb would do that :laughing: